app/models/payment.rb

```diff
@@ -32,7 +32,7 @@ def key
   end
 
   def bill_description
-    record['billReasonDescription']
+    Mylibrary::Application.config.bill_reasons[record['billReasonDescription']] || record['billReasonDescription']
```

Comment:
Can we also move this to a helper, so other instances could access this native value also?